packages/plugin-core/src/components/views/PreviewViewFactory.ts

```diff
@@ -30,4 +30,8 @@ export class PreviewPanelFactory {
 
     return PreviewPanelFactory._preview;
   }
+
+  static get preview() {
+    return PreviewPanelFactory._preview
```

Comment:
Let's not add a static getter for `PreviewProxy`, otherwise it'll be hard to manage coupling around the preview panel later (we had this problem in the past).  Instead, in `_extension.ts` you can create a single object and share it between the implementations of the TOGGLE_PREVIEW_LOCK and TOGGLE_PREVIEW commands

Reply:
I was uncomfortable with this change, but did not take time to think about it more. Thank you for pointing this out, thats a good way to solve this.

[`b147ea1` (#3293)](https://github.com/dendronhq/dendron/pull/3293/commits/b147ea11f4742cd0a18334c7bc32913092d405b1)